scenarios/arcaflow/sysbench-cpu-hog/input.yaml

```diff
@@ -0,0 +1,8 @@
+kubeconfig: {}
+node_selector:
+  kubernetes.io/hostname: minikube-m03
```

Comment:
That is the bug that I'm waiting to fix in the Arca engine, kept here for debugging, will be removed before merging.